pkgs/build-support/docker/default.nix

```diff
@@ -209,7 +209,7 @@ rec {
 
       postMount = ''
         echo "Packing raw image..."
-        tar -C mnt --mtime=0 -cf $out .
+        tar -C mnt --mtime='@1' -cf $out .
```

Comment:
I think technically `@0` is the unix epoch, but as long as it's consistent this ought to be fine